.github/PULL_REQUEST_TEMPLATE/generic.md

```diff
@@ -0,0 +1,6 @@
+---
+name: General
+about: Change which doesn't fit any of the other categories
+labels: "yoast cs/qa"
+---
+
```

Comment:
Aren't a description, refs and test instructions always relevant for a PR? I'd expect those headings in the generic template.

Reply:
Not really. Take the following PRs as examples of PRs which would fit in the `generic` template and where - other than a description, but that is implied - nothing else is needed:
#127, #137, #139, #147

Reply:
Not really. Take the following PRs as examples of PRs which would fit in the `generic` template and where - other than a description, but that is implied - nothing else is needed:
#127, #137, #139, #147